src/Backend.cs

```diff
@@ -8,7 +8,12 @@ public class Backend
         /// <summary>
         /// Base domain for backend service
         /// </summary>
-        public const string BASE_URL = "group.cluster.jppuk.net";
-        //public const string BASE_URL = "10.0.2.2:52134";
+        public static string BASE_URL { get; set; }
```

Comment:
Right in assuming all clients will now need to set this? If so consider, the getter throwing an exception if null or empty?